src/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ include(BuildLGraphServer.cmake)
 include(BuildCypherLib.cmake)
 include(BuildClients.cmake)
 include(BuildCythonExtension.cmake)
+add_subdirectory(c)
```

Comment:
I recommend merging the cmakelists file of build-c-bindings into `BuildLGraphApi.cmake`, since c-bindings should be part of lgraph_api